### ISSUE TITLE
Where should I put a 3rd party Node.js connector

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/documentation#2529]
Hi,

So [this is cool](https://www.tentwentyfour.lu/2019/08/27/nextcloud-link/) but where should people find it? I have it now on nextcloud.com/developer but we're making some changes there and it is no longer a good fit. Where in the doc does this fit, anyone an opinion?

**Comment by skjnldsv** [comment]
I'm not sure we should. This is not officially supported, nor reviewed. :thinking: 

cc @nextcloud/javascript

**Comment by blizzz** [comment]
appstore integrations?

**Comment by juliushaertl** [comment]
I think it is fine to list them somewhere in the docs as long as we mark them as unofficial or 3rdparty. Github does that as well for example https://developer.github.com/v3/libraries/

**Comment by ChristophWurst** [comment]
Yep, let's allow these things in our docs, but with a clear note about the 3rdparty and unofficial aspect.

**Comment by skjnldsv** [comment]
The official docs aren't the right home for unofficial, unsupported third-party connectors it seems after all...
The Nextcloud App Store or the community hub (https://help.nextcloud.com) are probably better places to surface community projects like this ? 🙈
